### DISCUSSION TITLE
Welcome to jexec-build Discussions!

### DISCUSSION THREAD

**Discussion by saphirepankaj** [saphirepankaj/jexec-build#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by solanki08** [comment]
I have developed a simple standalone java application using Jdk 17 and I have followed all the steps you've provided to secure my jar. While running the new jar, it keeps on loading with no output. Can you please help ?

**Reply by saphirepankaj** [reply]
Can you share your simple standalone jar, I can check the issue, share file to saphire.pankaj@gmail.com.
